packages/tizen_notification/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## NEXT
 
+* Update example.
+
+## NEXT
```

Comment:
There should be only one `## NEXT` in this file.

Please consider publishing the package (0.1.1).